default/methods/task/task/task_user_modify.py

```diff
@@ -0,0 +1,108 @@
+from methods.regular.regular_api import *
```

Comment:
This one looks great Vitalii, I think you now have the grasp of how the backend is structured.

The only comment is the same as with the previous, to keep the api and core logic separated. I created a small commit as an example of how would I refactor it here: https://github.com/diffgram/diffgram/pull/451/commits/ef3e1285c03bbf26e419f1a0b67b077f1637fa2d

You can use this one to make the refactor to the `task_user_add` API